bitnami/kafka/values-production.yaml

```diff
@@ -271,14 +278,26 @@ auth:
   ## MANDATORY when method is 'sasl', or 'sasl_tls'
   ##
   jaas:
-    ## Kafka client user
+    ## Kafka client user. If specified clientUsers takes precedence.
     ##
     clientUser: user
 
-    ## Kafka client password
+    ## Kafka client user list
+    ##
+    ## clientUsers: user1,user2
+    ##
+    clientUsers: ""
```

Comment:
It should be an array, since it's easier to manage.
You can convert then to string when setting the env var using:
```
{{ join "," .Values.auth.jaas.clientusers | quote }}
```